src/constants.js

```diff
@@ -64,6 +64,15 @@ module.exports = {
   ALGOLIA_JS_URL: 'https://cdn.jsdelivr.net/npm/docsearch.js@2/dist/cdn/docsearch.min.js',
   ALGOLIA_INPUT_SELECTOR: '#algolia-search-input',
 
+  // src/plugins/copyCode.js
+  COPIED_TO_CLIPBOARD: 'Copied!',
```

Comment:
Since we use these text only for display purposes, and will not be reusing them anywhere else, is it necessary for us to put this in the constants file?

Reply:
Fair enough. Will move them out 👍